### DISCUSSION TITLE
@expo/config-plugins@6.0.1 does not support react-native-firebase

### DISCUSSION THREAD

**Discussion by shubham-ghanghotia** [invertase/react-native-firebase#7028] [General]
@react-native-firebase/app@17.4.2
│ └── @expo/config-plugins@5.0.4
├─┬ @react-native-firebase/auth@17.4.2
│ └── @expo/config-plugins@5.0.4
└─┬ expo@48.0.10
  ├─┬ @expo/cli@0.6.2
  │ ├── @expo/config-plugins@6.0.1 deduped
  │ └─┬ @expo/prebuild-config@6.0.0
  │   └── @expo/config-plugins@6.0.1 deduped
  ├── @expo/config-plugins@6.0.1
  └─┬ @expo/config@8.0.2
    └── @expo/config-plugins@6.0.1 deduped
I'm facing this isses, 
i don't know how can i fix it.

**Comment by mikehardy** [comment]
We haven't resolved this yet #6679 - workaround available here #6948 / resolving duplicate
